source/adapters/opencl/common.hpp

```diff
@@ -226,6 +236,58 @@ cl_int(CL_API_CALL *)(cl_command_queue queue, cl_program program,
                       cl_uint num_events_in_waitlist,
                       const cl_event *events_waitlist, cl_event *event);
 
+using clCreateCommandBufferKHR_fn = CL_API_ENTRY cl_command_buffer_khr(
```

Comment:
Rather than defining these ourselves, can we use the symbols from the OpenCL-Headers instead? https://github.com/KhronosGroup/OpenCL-Headers/blob/main/CL/cl_ext.h#L102-L110

Not sure what version of the headers you're testing, with but these should have been in for a while

Reply:
Discussed this offline. Leaving this as is to be safe for the Github CI.